workflow/rules/datavzrd.smk

```diff
@@ -116,5 +77,8 @@ rule go_enrichment_datavzrd:
         ),
     log:
         "logs/datavzrd-report/go_enrichment-{model}/go_enrichment-{model}_{gene_fdr}.go_term_fdr_{go_term_fdr}.log",
+    params:
+        offer_excel=lookup(within=config, dpath="report/offer_excel", default="false"),
```

Comment:
```suggestion
        offer_excel=lookup(within=config, dpath="report/offer_excel", default=False),
```

Reply:
Already done in response to previous comment.